src/azure.rs

```diff
@@ -395,5 +547,6 @@ mod tests {
         put_get_delete_list(&integration).await.unwrap();
```

Comment:
I think there were some exlusions in this test for azure due to lack of consistent error handling, these might be able to be removed now :smile: